docs/src/12_glossary.adoc

```diff
@@ -15,5 +15,8 @@ ifndef::imagesdir[:imagesdir: ../images]
 |Java Spring Boot (Spring Boot) is a tool that makes developing web application and microservices with Spring Framework faster and easier through three core capabilities: Autoconfiguration, an opinionated approach to configuration, the ability to create standalone applications.
```

Comment:
could we get rid of the references to technologies such as spring boot and postgres in our glossary? They are not necessary